### ISSUE TITLE
IQR as a metric of excluding mice

### ISSUE THREAD

**Issue by davidhbrann** [dattalab/moseq2-app#37]
What validation was done to indicate that `get_iqr_anomaly_sessions` correctly identifies outlier sessions and that the IQR across all frames is a reasonable metric? 

If it's either too stringent or loose, we'll then put out warnings when there shouldn't be or fail to identify outliers that are actually present.

There are also explicit methods for outlier detection (e.g. https://scikit-learn.org/stable/modules/generated/sklearn.covariance.EllipticEnvelope.html) that could be fit to frames or to the average mouse values. Perhaps these would be more appropriate?
